### ISSUE TITLE
The laser cut knob on Sheila's vertical adjustment mechanism is broken

### ISSUE THREAD

**Issue by JackiePease** [DoESLiverpool/somebody-should#1734]
This was made out of layers of glued together ply. The top few layers are missing. @magman2112 suggests that @huffeec might have cutting files?

**Comment by huffeec** [comment]
The tool came with the machine from Just Add Sharks. There is nothing on their [forums](https://community.andmirrors.co.uk/)  but Dominic might still have the file. 

From memory it's just an inkscape rounded cornered star shape, might be quicker just to replicate it?

**Comment by huffeec** [comment]
A 1/4in 12 point socket fits, I've left one in the machine for bed adjustment for now.

**Comment by amcewen** [comment]
I've ordered one of [these t-bars](https://www.ebay.co.uk/itm/274974274323) and a [1/4" 12-point socket](https://www.ebay.co.uk/itm/115458863266) to go with it.

**Comment by amcewen** [comment]
The socket has turned up.

**Comment by amcewen** [comment]
So, the socket was the correct size but didn't fit the bed adjuster (which is a 1/4" square drive).  However, I've found the one that @huffeec had initially left on it, which does fit.  (That also has "1/4" stamped into it, but is larger than the 1/4" socket I ordered, and one of mine that I brought in to check against).

So I've put the new - correctly labelled - 1/4" socket into the tools drawer, and the one that fits is now combined with the red t-bar and left with Sheila for use when adjusting the bed.

![DSC_1822](https://user-images.githubusercontent.com/108468/235941835-2e9cd622-b783-4e78-ad44-7ea002c6943e.JPG)
